vipps-ecom-api.md

```diff
@@ -395,6 +395,7 @@ If Vipps is installed, Vipps will automatically be opened.
    as this depends on user actions, network connectivity/speed, etc.
    Because of this, it is not possible to base an integration on a specific
    sequence of events.
+5. Callback and fallback URLs _must_ use HTTPS.
```

Comment:
slight over precision on fallback here

Reply:
Even more overprecision now:
---
The `callbackPrefix` URL _must_ use HTTPS.
The `fallBack` URL must use either HTTPS or a custom URL scheme (`myapp://`).